src/pages/messages/components/MessageList.tsx

```diff
@@ -47,22 +53,26 @@ const MessageItem = ({ message }: MessageItemProps) => {
 
   return (
     <li>
-      <Typography sx={{ display: 'flex', gap: '0.25rem' }}>
-        <span>
-          {senderQuery.isLoading ? (
-            <Skeleton sx={{ width: '8rem' }} />
-          ) : (
-            <b data-testid={dataTestId.registrationLandingPage.tasksPanel.messageSender}>
-              {senderName ? senderName : <i>{t('common.unknown')}</i>}
-            </b>
-          )}
-        </span>
-        <span data-testid={dataTestId.registrationLandingPage.tasksPanel.messageTimestamp}>
-          ({new Date(message.createdDate).toLocaleString()})
-        </span>
-      </Typography>
+      <Box sx={{ bgcolor: ticketColor[ticketType], p: '0.5rem', borderRadius: '4px', maxWidth: '20rem' }}>
+        <Typography flexWrap="wrap" sx={{ display: 'flex', justifyContent: 'space-between' }}>
+          <span>
+            {senderQuery.isLoading ? (
+              <Skeleton sx={{ width: '8rem' }} />
+            ) : (
+              <b data-testid={dataTestId.registrationLandingPage.tasksPanel.messageSender}>
+                {senderName ? senderName : <i>{t('common.unknown')}</i>}
+              </b>
+            )}
+          </span>
+          <span data-testid={dataTestId.registrationLandingPage.tasksPanel.messageTimestamp}>
+            {new Date(message.createdDate).toLocaleDateString()}
+          </span>
+        </Typography>
+
+        <Divider sx={{ mb: '0.5rem', bgcolor: '#B4A98E' }} />
```

Comment:
vi har en farge i `palette.divider` som er default for Divider. Sett om Divider skal ha ulike farge rundt omkring? **Hvis** den skal ha samme farge overalt kan fargen settes i theme i stedet

Reply:
Dividere i venstremeny er en annen farge, men kom frem til at vi bruker `primary.main` etter en samtale med Cecilie.